packages/components/src/components/FormSelect/FormSelect.tsx

```diff
@@ -36,12 +36,10 @@ const FormSelect = React.forwardRef<HTMLButtonElement, FormSelectProps>(
           required={required}
           {...props}
         />
-        {helpText ? (
+        {helpText && (
```

Comment:
I agree that we need to fix all of them. But don't you think it would be better to do it iteratively? 
We already have an inconsistency in the apps because some of us decided not to use these fields just because of this extra space 🤷🏻‍♀️ 

I can change all of them in this PR and then update the apps. But it will take more time and will impact more parts of the code.

Reply:
I think we should discuss it with the team. If we remove it, and someone uses the helptext, then the height will be increased and effect everything.

I think it is better to go slow, take our time and be consistent rather than make iterative adjustments that will create inconsistencies.

Reply:
I think both approaches are fine. We should make it consistent ASAP but that does not mean we need to do it all in one PR, so doing it iterative is also fine, as long as the iterations don't take weeks.